gen/templates/model.go

```diff
@@ -109,7 +113,11 @@ func (data {{camelCase .Name}}) toBody(statusReplace bool) nxos.Body {
 	{{- range .ChildClasses}}
 	{{- $childClassName := .ClassName }}
 	{{- if eq .Type "single"}}
+	{{- if len .Attributes}}
 	attrs = ""
+	{{- else}}
+	attrs = "{}"
```

Comment:
If we want it to be a map, I think we can always make it an empty map and don't need a length condition.